gitless/cli/gl_remote.py

```diff
@@ -76,3 +81,19 @@ def _do_delete(delete_r, remotes):
       pprint.err('Remote \'{0}\' doesn\'t exist'.format(r))
       errors_found = True
   return not errors_found
+
+
+def _do_rename(rename_r, remotes):
+  errors_found = False
+  if len(rename_r) != 2:
+    pprint.err(
+        'Expected 2 arguments to gl remote -rn')
```

Comment:
add that you expect it to be in the format "current remote name" and then "new remote name"?